maintenance_page/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM nginx:1.25.4
```

Comment:
Could we try the unprivileged image as in [the tech guidance](https://github.com/DFE-Digital/technical-guidance/blob/master/Dockerfile#L1) ?
If it works, we can update the template

Reply:
added the image